python/ray/autoscaler/_private/resource_demand_scheduler.py

```diff
@@ -179,18 +183,32 @@ def get_nodes_to_launch(
             return self._legacy_worker_node_to_launch(
                 nodes, launching_nodes, node_resources,
                 resource_demands + request_resources_demands)
-        placement_group_nodes_to_add, node_resources, node_type_counts = \
+
+        spread_pg_nodes_to_add, node_resources, node_type_counts = \
```

Comment:
Should it just be `pg_nodes_to_add`? Spread is a type of placement group right?
```suggestion
        placement_group_nodes_to_add, node_resources, node_type_counts = \
```

Reply:
There are two placement groups and I used the current naming to make it clearer. Originally we counted only spreads in placement_group_nodes_to_add, which is confusing because placement_group_nodes_to_add should be both. The placement groups nodes to add are actually the combination of spreads and placement group demands.